### ISSUE TITLE
(Add) Directv Uruguay Guide Please 

### ISSUE THREAD

**Issue by tuvasasa** [iptv-org/epg#1043]
### Source

https://www.directv.com.uy/guia/ 

### Notes

Hello, i would like a petition
I have a request, basically I would like to add the epg of Directv Uruguay because it has the epg of several channels from Uruguay, unlike the Argentine version.
I also have a correction in the EPG logos, "TVNacionalUruguay.uy" is no longer called "Television Nacional", now its name is "Canal 5".
This is a New logo: "https://dnqt2wx2urq99.cloudfront.net/ondirectv/LOGOS/Canales/UY/191.png"
Thanks for this page, you are awesome and see later

**Comment by LaneSh4d0w** [comment]
Cannot access the website from my French IP.

**Comment by LaneSh4d0w** [comment]
Resolved with #1047.

**Comment by freearhey** [comment]
Link to the new guide:

```
https://iptv-org.github.io/epg/guides/uy/directv.com.uy.epg.xml.gz
```
